### PR TITLE
New utils

### PR DESCRIPTION
**Pull request by pochretien**
add transparency utils for each variable
add utils for the hover with each color in the palette
add specific classes for the card

### CODE REVIEW

**Comment by pochretien** [comment]
Examples incoming

File: scss/utility/layout.scss [hunk 1]
```diff
@@ -155,3 +155,11 @@
 .small-desktop-breakpoint-responsive-UI {
   min-width: $breakpoint-desktop-small;
 }
+
+@for $i from 1 through length($transparencies) {
+  .mod-transparency-#{$i} {
```
Comment (GermainBergeron) [inline scss/utility/layout.scss]: I'm not sure layout is the best place to put this but I don't really mind

I would prefer to call it `transparency-X` (without mod-)

File: scss/variables.scss [hunk 2]
```diff
@@ -528,6 +528,10 @@ $ribbon-font-size: 0.8em;
 $ribbon-letter-spacing: 1px;
 $ribbon-rotation-angle: 45deg;
 
+// Card
+$card-container-box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.3);
```
Comment (GermainBergeron) [inline scss/variables.scss]: `$card-container-box-shadow: 0 2px 8px 0 rgba($black, $transparency-4);` (I'm not sure which number it should be)

**Comment by pochretien** [comment]
image
image
image

**Pull request merged**